R/population-study-design.R

```diff
@@ -0,0 +1,226 @@
+#' @title addStudyParameters
+#' @param population `Population` object
+#' @param studyDesignFile file name of study design table
+#' @export
+#' @import ospsuite
+addStudyParameters <- function(population, studyDesignFile = NULL) {

```

Comment:
I am a bit confused with the signature. If someone calls addStudyDesign, shouldn't the file exist?
Or do you intend to call it no matter what?

Reply:
From what I understand this function may be called mostly from within a workflow.
So, I was thinking of using the function no matter what the user provides to a simulation set.
But it may be better to split between an actual addStudyParameter and a workflow specific function that will used a simulation set as an input and will check if the file is provided.